nix/build.nix

```diff
@@ -0,0 +1,18 @@
+{
+  base,
+}:
+let
+  niv = import ./sources.nix;
+  nixpkgsSrc = niv.nixpkgs;
```

Comment:
`nixpkgsSrc` isn't used, so you could remove it

Reply:
:+1: